c_glib/arrow-glib/compute.cpp

```diff
@@ -37,6 +37,8 @@
 #include <arrow/acero/exec_plan.h>
 #include <arrow/acero/options.h>
 
+auto registration_status_ = arrow::compute::RegisterComputeKernels();
```

Comment:
How about using `Initialize()` instead of `RegisterComputeKernels()`? We may want to do more on load.

Reply:
what do you think about `InitializeArrowCompute()` or `InitializeComputeKernels`? 
On places like pyarrow we lose the namespace and it would be good to have more context on what is being initialized. If yes, which one do you prefer?

Reply:
> On places like pyarrow we lose the namespace

Why? It seems that PyArrow doesn't use `using arrow::compute`.

> If yes, which one do you prefer?

I prefer `::arrow::compute::Initialize()` (or `arrow::compute::InitializeCompute`).

`Arrow` (and `Compute`) in `arrow::compute::InitializeArrowCompute()` are redundant.

If we want to initialize not only kernels but also others, `InitializeComputeKernels()` will be misleading (or users need to call `InitializeComputeKernels()` and `InitializeSomething()`.)

Reply:
you are correct, I've moved to `::arrow::compute::Initialize()`